public/javascripts/login.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * Created by Trevor on 6/26/2015.
+ */
+function sendAuth() {
+
+    $(document).ready(function () {
+        var user, pass;
+
+        user = $("#username").val();
+        pass = $("#password").val();
+        var host = window.apiRoute + "/login/";
+
+        $.post(host, {user: user, password: pass},
+            function (data) {
+                var cookie = data; //req.cookies.auth
+
+                if (data != "Invalid Credentials!") {
+                    var redisInput = window.apiRoute + '/redis/SetState/' + cookie + '/' + user + '/' + "login.html";
```

Comment:
Please move this out of the front end, it is not the front end's job to do this, the login route should be setting the cookie.